packages/lib-user/src/components/UserStats/components/myStyled.js

```diff
@@ -0,0 +1,6 @@
+import styledComponents from 'styled-components';
+
+const styled = typeof styledComponents === 'function' ? styledComponents : styledComponents.default;
+export const css = styledComponents.css;
```

Comment:
This file will need to re-export all of the styled components API. That seems hard to maintain.

Reply:
Agreed!